en/gps_compass/rtk_gps.md

```diff
@@ -46,6 +46,7 @@ Device | GPS | Compass | [DroneCAN](../dronecan/index.md) | [GPS Yaw](#configuri
 [LOCOSYS Hawk R1](../gps_compass/rtk_gps_locosys_r1.md) | MC-1612-V2b | | | |
 [LOCOSYS Hawk R2](../gps_compass/rtk_gps_locosys_r2.md) | MC-1612-V2b | IST8310 | | |
 [mRo u-blox ZED-F9 RTK L1/L2 GPS](https://store.mrobotics.io/product-p/m10020d.htm) | F9P | ✓ | | [Dual F9P][DualF9P] |
+[Navisys L1/L2 ZED-F9P RTK Base](https://www.navisys.com.tw/productdetail?name=GR901&class=RTK) | F9P | | | |
```

Comment:
This link gives me an internal server error, and also asks for me to select a certificate. Does it normally work?
Have you considered creating a page for connecting/using your receiver?

Reply:
OK, link works now.

Does this come with cables/connectors for connecting to Pixhawk? My take on this is that probably no and you are JUST offering this as a base station right?

Reply:
Thank you. I'll just make it clear this is "base only".
```suggestion
[Navisys L1/L2 ZED-F9P RTK - Base only](https://www.navisys.com.tw/productdetail?name=GR901&class=RTK) | F9P | | | |
```